packages/vite/src/ember.ts

```diff
@@ -107,7 +107,7 @@ function minification(config: UserConfig, mode: string) {
    */
   config.build ||= {};
 
-  if (!config.build.minify) {
+  if (config.build.minify === undefined || config.build.minify === true) {
     config.build.minify = 'terser';
```

Comment:
Need to make sure that `minify: false` isn't overwritten with `'terser'`

The option technically allows boolean so I've catered for `minify: true` as well, to apply 'terser'
https://vite.dev/config/build-options#build-minify

Everything else will be handed off to vite (false, 'esbuild')